x-pack/test/functional/apps/maps/group2/embeddable/dashboard.js

```diff
@@ -100,16 +99,10 @@ export default function ({ getPageObjects, getService }) {
     });
 
     it('should apply new container state (time, query, filters) to embeddable', async () => {
-      await filterBar.selectIndexPattern('logstash-*');
-      await filterBar.addFilter('machine.os', 'is', 'win 8');
+      await filterBar.addFilterAndSelectDataView('logstash-*', 'machine.os', 'is', 'win 8');
       await PageObjects.maps.waitForLayersToLoad();
 
-      // retry is fix for flaky test https://github.com/elastic/kibana/issues/113993
-      // timing issue where click for addFilter opens filter pill created above instead of clicking addFilter
-      await retry.try(async () => {
-        await filterBar.selectIndexPattern('meta_for_geo_shapes*');
-        await filterBar.addFilter('shape_name', 'is', 'alpha'); // runtime fields do not have autocomplete
-      });
+      await filterBar.addFilterAndSelectDataView('meta_for_geo_shapes*', 'shape_name', 'is', 'alpha');
```

Comment:
Able to remove retry because search bar redesign has moved add filter button so its no longer next to newly created filter pills.